test/e2e_node/remote/ssh.go

```diff
@@ -122,3 +125,16 @@ func runSSHCommand(cmd string, args ...string) (string, error) {
 	}
 	return string(output), nil
 }
+
+// getPrivateSSHKey returns the path to ssh private key
+func getPrivateSSHKey() string {
```

Comment:
```suggestion
func privateSSHKeyPath() (string, error) {
```

Reply:
Check the existence here too? Then we can easily move this check earlier later, and just pass the path down

Reply:
@endocrimes I've updated the PR so would you review it?